analyzer/src/funTest/assets/projects/synthetic/dotnet-expected-output-with-nuspec.yml

```diff
@@ -569,57 +265,17 @@ packages:
     url: ""
     revision: ""
     path: ""
-- id: "NuGet::System.Threading.Tasks.Extensions:4.5.4"
-  purl: "pkg:nuget/System.Threading.Tasks.Extensions@4.5.4"
-  authors:
-  - "Microsoft"
-  declared_licenses:
-  - "https://github.com/dotnet/corefx/blob/master/LICENSE.TXT"
-  declared_licenses_processed:
-    spdx_expression: "MIT"
-    mapped:
-      https://github.com/dotnet/corefx/blob/master/LICENSE.TXT: "MIT"
-  description: "Provides additional types that simplify the work of writing concurrent\
-    \ and asynchronous code.\n\nCommonly Used Types:\nSystem.Threading.Tasks.ValueTask<TResult>\n\
-    \ \n7601f4f6225089ffb291dc7d58293c7bbf5c5d4f"
-  homepage_url: "https://dot.net/"
+- id: "NuGet::foobar:1.2.3"
```

Comment:
@sschuberth same answer as above https://github.com/oss-review-toolkit/ort/pull/6209#discussion_r1115498090

> One cannot assume that a package is "dummy" just because it does not exist in a NuGet repository. It could be a local non-published package. Note that this is the same behavior that we have in the python-inspector: we return these packages but they do not have any public URLs and other related attributes retrieved from repository.